cli/internal/cmd/upgradeapply.go

```diff
@@ -305,7 +330,14 @@ func (u *upgradeApplyCmd) upgradeAttestConfigIfDiff(cmd *cobra.Command, newConfi
 	}
 
 	if !flags.yes {
-		ok, err := askToConfirm(cmd, "You are about to change your cluster's attestation config. Are you sure you want to continue?")
+		cmd.Println("The configured attestation config is different from the attestation config in the cluster.")
+		diffStr, err := diffAttestationCfg(clusterAttestationConfig, newConfig)
+		if err != nil {
+			return fmt.Errorf("diffing attestation configs: %w", err)
+		}
+		cmd.Println("The following changes will be applied to the attestation config:")
+		cmd.Println(diffStr)
```

Comment:
I guess this would be printed, even if there is no diff?

Reply:
no, because of https://github.com/edgelesssys/constellation/blob/9ae89133265b835c72b8d8fb62127df7a0d5f686/cli/internal/cmd/upgradeapply.go#L328 it returns early